The vtk io tests are failing in windows

I am getting the following error in windows:

```
======================================================================
ERROR: test_ascii_vtk_output (test_vtk_output_process.TestVtkOutputProcess)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_vtk_output_process.py", line 16, in test_ascii_vtk_output
    ExecuteBasicVTKoutputProcessCheck("ascii")
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_vtk_output_process.py", line 158, in ExecuteBasicVTKoutputProcessCheck
    os.path.join("vtk_output_process_ref_files", file_format, "Main_0_"+str(step)+".vtk"))
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_vtk_output_process.py", line 106, in Check
    }""")
RuntimeError: [json.exception.parse_error.101] parse error at line 2, column 37: syntax error while parsing value - invalid string: forbidden character after backslash; last read: '"E:\G'

======================================================================
ERROR: test_binary_vtk_output (test_vtk_output_process.TestVtkOutputProcess)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_vtk_output_process.py", line 19, in test_binary_vtk_output
    ExecuteBasicVTKoutputProcessCheck("binary")
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_vtk_output_process.py", line 158, in ExecuteBasicVTKoutputProcessCheck
    os.path.join("vtk_output_process_ref_files", file_format, "Main_0_"+str(step)+".vtk"))
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_vtk_output_process.py", line 106, in Check
    }""")
RuntimeError: [json.exception.parse_error.101] parse error at line 2, column 37: syntax error while parsing value - invalid string: forbidden character after backslash; last read: '"E:\G'

======================================================================
FAIL: test_ModelPart_str (test_object_printing.TestObjectPrinting)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "E:\GitLab-Runner\builds\ZgfosLiz\0\poesis\Poesis\bin\Windows\Release\kratos\tests\test_object_printing.py", line 91, in test_ModelPart_str
    self.assertMultiLineEqual(str(model_part), model_part_str)
AssertionError: '-Mai[327 chars]    -Inlets- model part\n        Number of tab[1258 chars] 0\n' != '-Mai[327 chars]    -Outlet- model part\n        Number of tab[1258 chars] 0\n'
  -Main- model part
      Buffer Size : 1
      Number of tables : 1
      Number of sub model parts : 2
      Current solution step index : 0
  
      Mesh 0 :
          Number of Nodes       : 6
          Number of Properties  : 1
          Number of Elements    : 4
          Number of Conditions  : 5
          Number of Constraints : 0
  
+     -Outlet- model part
+         Number of tables : 0
+         Number of sub model parts : 0
+ 
+         Mesh 0 :
+             Number of Nodes       : 0
+             Number of Properties  : 1
+             Number of Elements    : 0
+             Number of Conditions  : 1
+             Number of Constraints : 0
      -Inlets- model part
          Number of tables : 1
          Number of sub model parts : 2
  
          Mesh 0 :
              Number of Nodes       : 3
              Number of Properties  : 0
              Number of Elements    : 1
              Number of Conditions  : 3
              Number of Constraints : 0
+         -Inlet2- model part
+             Number of tables : 0
+             Number of sub model parts : 0
+ 
+             Mesh 0 :
+                 Number of Nodes       : 0
+                 Number of Properties  : 0
+                 Number of Elements    : 0
+                 Number of Conditions  : 2
+                 Number of Constraints : 0
          -Inlet1- model part
              Number of tables : 0
              Number of sub model parts : 0
  
              Mesh 0 :
                  Number of Nodes       : 2
                  Number of Properties  : 0
                  Number of Elements    : 0
                  Number of Conditions  : 2
                  Number of Constraints : 0
-         -Inlet2- model part
-             Number of tables : 0
-             Number of sub model parts : 0
- 
-             Mesh 0 :
-                 Number of Nodes       : 0
-                 Number of Properties  : 0
-                 Number of Elements    : 0
-                 Number of Conditions  : 2
-                 Number of Constraints : 0
-     -Outlet- model part
-         Number of tables : 0
-         Number of sub model parts : 0
- 
-         Mesh 0 :
-             Number of Nodes       : 0
-             Number of Properties  : 1
-             Number of Elements    : 0
-             Number of Conditions  : 1
-             Number of Constraints : 0


----------------------------------------------------------------------
Ran 219 tests in 17.273s
```

The first errors are due to the back slashes `"\"` in the path  of `__file__` 

Is anybody had the same issue?

--- a/kratos/tests/test_vtk_output_process.py
+++ b/kratos/tests/test_vtk_output_process.py
@@ -101,9 +101,11 @@ def Check(output_file,reference_file):
 
     ## Settings string in json format
     params = KratosMultiphysics.Parameters("""{
-        "reference_file_name" : \"""" + GetFilePath(reference_file) + """\",
-        "output_file_name"    : \"""" + output_file + """\"
+        "reference_file_name" : "", 
+        "output_file_name"    : ""
     }""")
+    params["reference_file_name"].SetString(GetFilePath(reference_file))
+    params["output_file_name"].SetString(output_file)
 
     cmp_process = compare_process.CompareTwoFilesCheckProcess(params)
 

